int8 controller_to_execute  # controller enum, defined in gps_pb2

CaffeParams caffe
LinGaussParams lingauss
TfParams tf
